build/yamls/base/conf/antrea-agent.conf

```diff
@@ -93,6 +93,10 @@ wireGuard:
 #  The port for WireGuard to receive traffic.
 #  port: 51820
 
+egress:
```

Comment:
The windows base conf also need to update: build/yamls/windows/base/conf/antrea-agent.conf

Reply:
In the build/yamls/windows/base/conf/antrea-agent.conf, there is no `Egress` in the featureGates, so I didn't modify this file.

Reply:
Yes, Egress feature is not supported on Windows yet: https://github.com/antrea-io/antrea/blob/main/docs/feature-gates.md#requirements-for-this-feature-6